src/pages/RequestCallPage.js

```diff
@@ -274,6 +285,8 @@ class RequestCallPage extends Component {
     }
 
     render() {
+        const isBlockedFromConcierge = !_.isEmpty(this.props.blockedFromConcierge) && User.isBlockedFromConcierge(this.props.blockedFromConcierge.expiresAt);
```

Comment:
NAB: You could replace `!_.isEmpty(this.props.blockedFromConcierge)` with just `this.props.blockedFromConcierge.expiresAt` since that would be undefined in the default case.

Reply:
Hm, yeah. I think I like this better since it'll explicitly set to false rather than undefined.